static/locales/fr/translation.json

```diff
@@ -112,12 +112,12 @@
 			"title": "Dépannage",
 			"restore": "Restaurer les paramètres par défaut",
 			"restore_warning": "Cela réinitialisera TOUS les paramètres avec leurs valeurs par défaut.",
-			"warning": "Non disponible comme hôte dans le jeu!"
+			"warning": "Non disponible en tant qu'hôte dans le jeu!"
 		}
 	},
 	"lobbybrowser": {
-		"header": "Lobbies Publiques",
-		"code": "Code du Lobby",
+		"header": "Parties publiques",
```

Comment:
It's a lobby (the place that people can go) not a party/group (the people in a lobby)

Reply:
Plus in other places you left it as a lobby, try and maintain consistency